Assets/VRTK/Scripts/VRTK_Hip_Tracking.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace VRTK
```

Comment:
Need to include the header comment line, stating script name, category and list position (check another script for an example)